docs/Architecture Decision Record/009-assign-superuser.md

```diff
@@ -0,0 +1,23 @@
+# 9. Assigning the Superuser
+Date: 2021-02-15
+
+## Status
+
+Accepted
+
+## Context
+
+Usually in Django we would assign the first superuser through the CLI, but because this will not
+be available in production, we will need another method.
```

Comment:
One thought -- it might be worth it to mention or describe the "why" behind this. Cloud.gov has a nice documentation writeup about why this is a recommended security and compliance practice that we could link to, and/or borrow from:

https://cloud.gov/docs/deployment/production-ready/#prevent-non-auditable-changes-to-production-apps

* SSH access makes changes harder to audit
* Relates to AC-17
* Additionally, relates to security concept of least privilege -- SSH access as root is extremely high-privilege